tools/install_mac_deps.sh

```diff
@@ -41,12 +42,14 @@ BOOST_LIB="boost@1.85"
 REQUIRED_PACKAGES=(
     autoconf automake ccache cmake libtool
     openssl pkg-config coreutils bison grep ninja
-    ${BOOST_LIB}
 )
 for package in "${REQUIRED_PACKAGES[@]}"; do
   brew_install ${package}
 done
 
+# The boost installation is flaky, do not fail if it fails.
+brew_install ${BOOST_LIB} || echo "Failed to install ${BOOST_LIB}"
```

Comment:
But how does the build work without boost?

Reply:
The problem is that boost is installed fine but the installation of a dependency of that brew package fails (Python 3.12). Unclear to me how to fix that. Maybe there is a flag for that problem.